### PR TITLE
Feat:Added List and Input component

### PR DESCRIPTION
**Pull request by neetukumari4858**
Feat:Added List and Input component
list
   1.Spaced (normal lists)
   2.Stacked (Notification Page)
Input
  1.Textbox
  2.with validation and errors style

### CODE REVIEW

File: components/Alert.css [hunk 1]
```diff
@@ -93,7 +93,7 @@ body{
     height: 28rem;
     background-color:gray;
 }
-.Anchor_link{
+.Anchor-link{
```
Comment (ajit-sharmaGH) [inline components/Alert.css:96]: I think using small letters for classes will be more compatible

File: components/Input.css [hunk 2]
```diff
@@ -0,0 +1,88 @@
+@import url("/Navigation.css");
+@import url("/document.css");
+.Input{
```
Comment (ajit-sharmaGH) [inline components/Input.css:3]: here also

File: components/List.html [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+                    style="width: 588px; height: 409px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (ajit-sharmaGH) [inline components/List.html:77]: you can style iframe using external styling

**Comment by ajit-sharmaGH** [comment]
Hey, I went through your code, everything looks fine to me; as a suggestion I commented there!

**Review by ajit-sharmaGH — APPROVED** [review]

File: components/Input.css [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+}
+.container{
+    width: 66rem;
+    height: 34.5rem;
+    margin-top: -69.2rem;
+    margin-left: 15.1rem;
+    overflow-y: scroll;
+    overflow-x: hidden;
+}
+p{
```
Comment (Bhakti199) [inline components/Input.css]: use class here

File: components/Input.css [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+.mail_input{
+    margin-top: -1.5rem;
+    padding: 0.8rem 5.9rem;
+    border-radius: 0.5rem;
+    border: 2px solid rgb(3, 202, 3) ;
+}
+.password_outer_div{
+    height: 10rem;
+    width: 27rem;
+    box-shadow: 1px 1px 2px 3px gainsboro;
```
Comment (Bhakti199) [inline components/Input.css:57]: assign gainboro to a variable in root and then use that variable here

File: components/Input.css [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+   
+}
+.password_inner_div{
+    height: 6rem;
+    width: 23.2rem;
+    color:red;
```
Comment (Bhakti199) [inline components/Input.css:63]: here too

File: components/List.css [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+    margin-left: 2rem;
+    box-shadow: 1px 1px 1px 3px gainsboro;
+}
+.list-title{
+    width: 12rem;
+    height: 2.2rem;
+    list-style: none;
+    display: flex;
+    flex-direction: column;
+    justify-content: end;
```
Comment (Bhakti199) [inline components/List.css:29]: you may define utility classes for this in main css file

File: components/List.html (hunk 3 above)
Comment (Bhakti199) [inline components/List.html:77]: use `rem` instead of `px`

**Review by Bhakti199 — COMMENTED** [review]
overall code is nicely written, below are some suggestions from my side.

**Review by Bhakti199 — APPROVED** [review]
good to merge.

**Pull request merged**